docs/topics/con_purpose-of-the-health-check-pattern.adoc

```diff
@@ -0,0 +1,13 @@
+[id='purpose-of-the-health-check-pattern_{context}']
+= Purpose of the Health Check Pattern
```

Comment:
Titles should be in sentence case per a style decision from CCS

Reply:
Fixed throughout.